tests/test_users.py

```diff
@@ -184,7 +185,7 @@ def test_roles(self):
 
         with self.subTest("A user inherits multiple role policies when the user has multiple roles."):
             self.assertListEqual(sorted(user.lookup_policies()),
-                                 sorted([user.statement] + role_statements))
+                                 sorted([user.statement, *self.default_user_policies, *role_statements]))
```

Comment:
Does the star operator have advantages over `+`, or is this a stylistic preference?

Reply:
it's slightly more performant and shorter to write

Reply:
95% stylistic